app/javascript/packs/spinner.js

```diff
@@ -0,0 +1,13 @@
+const loadSpinner = () => {
+  const importGistsBtn = document.getElementById("import-gists");
+  if (importGistsBtn) {
+    const spinner = `<div class="spinner-border text-light" role="status">
+                          <span class="sr-only">Loading...</span>
+                        </div>`;
+    importGistsBtn.addEventListener("click", () => {
+      importGistsBtn.innerHTML = spinner;
+    });
+  }
+}
+
+export { loadSpinner }
```

Comment:
no trailing line 🙈 
```suggestion
export { loadSpinner }

```

Reply:
My editor is configured to not add one.
Is this a convention thing?